layouts/default.vue

```diff
@@ -145,6 +145,9 @@
           { hreflang: 'x-default', rel: 'alternate', href: this.canonicalUrlWithoutLocale },
           ...i18nSeo.link
         ],
+        script: [
+          { src: `https://www.googleoptimize.com/optimize.js?id=${this.$config.googleOptimize.id}` }
+        ],
```

Comment:
This section should be optional depending on:
a) A google Optimize ID being present.
b) An experiment being active/available through `this.$exp`.
Otherwise we don't need to include the optimize code at all.